`Error: Cannot find module` when migrating to vitest

I tried to migrate a service to Vitest but I got this error : 

```
Error: Cannot find module '/Users/gabrielcolson/Documents/<...>/node_modules/serverless-spy/lib/src/ServerlessSpy' imported from /Users/gabrielcolson/Documents/<...>/node_modules/serverless-spy/lib/index.mjs
```

I found a workaround thanks to Cursor but I think this is an issue on ServerlessSpy's side: 
```ts
// vitest.config.ts
import { defineConfig } from "vitest/config"
import path from "path"

export default defineConfig({
  test: {
    typecheck: {
      tsconfig: "./tsconfig.dev.json",
    },
    setupFiles: [`${__dirname}/test/setup.ts`],
    include: ["test/e2e/**/*.e2e.ts"],
    poolOptions: {
      threads: {
        singleThread: true,
      },
    },
    testTimeout: 30000,
  },

  // added by cursor
  resolve: {
    alias: {
      "serverless-spy": path.resolve(__dirname, "node_modules/serverless-spy/lib/index.js"),
      "serverless-spy/lib/src/ServerlessSpy": path.resolve(
        __dirname,
        "node_modules/serverless-spy/lib/src/ServerlessSpy.js"
      ),
    },
  },
})
```

The same test works with Jest

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -29,7 +29,7 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: sudo docker
         run: sudo chmod 777 /var/run/docker.sock
       - name: Install dependencies
@@ -51,7 +51,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: Download build artifacts
         uses: actions/download-artifact@v4
         with:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -24,7 +24,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: sudo docker
         run: sudo chmod 777 /var/run/docker.sock
       - name: Set git identity
@@ -56,7 +56,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: Download build artifacts
         uses: actions/download-artifact@v4
         with:
@@ -82,7 +82,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: Download build artifacts
         uses: actions/download-artifact@v4
         with:

--- a/.github/workflows/upgrade-main.yml
+++ b/.github/workflows/upgrade-main.yml
@@ -18,7 +18,7 @@ jobs:
           ref: main
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: Install dependencies
         run: npm ci
       - name: Upgrade dependencies
@@ -41,7 +41,7 @@ jobs:
           ref: main
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
       - name: Download patch
         uses: actions/download-artifact@v4
         with:

--- a/package.json
+++ b/package.json
@@ -132,6 +132,7 @@
   },
   "main": "lib/index.js",
   "exports": {
+    "types": "./lib/index.d.ts",
     "import": "./lib/index.mjs",
     "require": "./lib/index.js"
   },

